openlibrary/plugins/openlibrary/js/check-ins/index.js

```diff
@@ -355,9 +355,19 @@ function closeDialog(workOlid) {
 function showDateView(workOlid, year, month, day) {
     let date = year
     if (month) {
-        date += `-${month}`
+        if (month.length<2){
+            month= month.padStart(2, '0')
+            date+= `-${month}`
+        } else {
+            date += `-${month}`
+        }
```

Comment:
```suggestion
        date += `-${month.padStart(2, '0')}`
```
Use `padStart` in a way that avoids extra conditionals.